portfolio/src/main/webapp/script.js

```diff
@@ -40,18 +44,52 @@ function showSlides() {
   for(slide of slides)
     slide.style.display = 'none';
 
-  slideIndex = slideIndex%slides.length; 
+  slideIndex = slideIndex % slides.length; 
   
   for(dot of dots)
     dot.className = dot.className.replace(" active","");
-
+  
   slides[slideIndex].style.display = "block";  
   dots[slideIndex].className += " active";
 }
 
 function showSlidesAuto()
 {
-    slideIndex+=1;
-    console.log(slideIndex);
+    slideIndex++;
     showSlides();
 }
+//change width css attribute
+function resizePhoto(photo,size)
+{
+    photo.style.width = size;
+}
+function changeSizeState(photo)
+{
+    sizeStateBig = !sizeStateBig;
+    if(sizeStateBig)
+        resizePhoto(photo,"75vw");
+    else
+        resizePhoto(photo,"60vw");
+}
+async function getData(path) {
+  // The fetch() function returns response object, and is asynchronous.
+  const response = await fetch(path);
+  //the text() function gets the text from the response asynchronously
+  const data = await response.text();
+  // When the request is complete return text from response
+  return data;
+}
+async function includeHTML(fileName)
+{
+    const element = document.getElementById(`include-${fileName}`);
+    if(element)
+    {
+        const header = await getData(`/${fileName}.html`);
```

Comment:
(Optional): To make it clearer that some of your HTML files are full-blown HTML pages and some are pieces of pages that are designed to be included, consider moving all of the partial pages to separate folder like `webapp/partials/` or `webapp/includes/`.